submodules/wizard/views/step-review.html

```diff
@@ -72,204 +72,214 @@
 {{/inline}}
 
 <div id="step_review" class="wizard-step">
-	<div class="step-title">
-		{{i18n.accountsApp.wizard.steps.review.title}}
+	<div class="flex-container">
+		<div class="step-title flex-fit">
+			{{i18n.accountsApp.wizard.steps.review.title}}
+		</div>
+		<button id="step_print" class="step-action-button flex-initial">
```

Comment:
I think just `monster-button` is the style you are looking for.

Reply:
Indeed, thanks!